ACE/ace/config-hpux-11.00.h

```diff
@@ -162,7 +162,6 @@
 #define ACE_HAS_CLOCK_SETTIME
 
 #define ACE_LACKS_CLOCK_MONOTONIC
-#define ACE_LACKS_MONOTONIC_TIME
```

Comment:
I couldn't find a reference to this macro anywhere other than here and in the monictonic code. Under my changes, `ACE_LACKS_CLOCK_MONOTONIC` should have the same effect. Regardless I kept a check for it on the chance a user passes it.

Reply:
Why not just remove it and mention this in the NEWS file?